Gems/ROS2/Code/Source/RobotImporter/Utils/FilePath.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "FilePath.h"
+
+namespace ROS2
+{
+    namespace Utils
+    {
+        AZStd::string GetCapitalizedExtension(const AZ::IO::Path& filename)
+        {
+            AZStd::string extension{ filename.Extension().Native() };
```

Comment:
I'm thinking that we can check if `filename` has an extension here, and if not, return an empty `""` string, otherwise return the capitalized extension.

Reply:
Resolved in https://github.com/RobotecAI/o3de-extras/pull/14.